core/container/intrusive/list/intrusive_list.odin

```diff
@@ -61,7 +81,13 @@ remove :: proc "contextless" (list: ^List, node: ^Node) {
 		}
 	}
 }
+/*
+Removes from the given list all elements that satisfy a condition.
```

Comment:
Implied, but given you mentioned O(1) complexity elsewhere, perhaps mention O(N) complexity here and for the contextless variant.